src/back/CrossTeamTestSuite/src/CrossTeamTestSuite/Endpoints/Converters/GetQueryParamsConverters/GetQueryParamConverter.cs

```diff
@@ -0,0 +1,89 @@
+using System.Reflection;
+using System.Text;
+using System.Text.Json.Serialization;
+using CrossTeamTestSuite.Endpoints.Contracts.Abstraction;
+using CrossTeamTestSuite.Endpoints.Converters.ValueConverters;
+
+namespace CrossTeamTestSuite.Endpoints.Converters.GetQueryParamsConverters;
+
+public class GetQueryParamConverter<TRequest>
+    where TRequest : class, IRequest
+{
+    private readonly List<object> valueConverters;
+
+    public GetQueryParamConverter()
+    {
+        valueConverters = new()
+        {
+            new StringValueConverter(),
+            new IntValueConverter(),
+        };
```

Comment:
You could probably make `ListValueConverter` to handle Lists automatically. I mean this situation:
`TaxIdTypesFilter=[0, 1, 2]` should be converted to `TaxIdTypesFilter=0&TaxIdTypesFilter=1&TaxIdTypesFilter=2`

Reply:
I can.